tpp-app/src/services/CalendarService.js

```diff
@@ -0,0 +1,90 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { KEYS, TRACK_SYMPTOMS } from './utils/constants.js'
+import { Symptoms } from './utils/models.js';
+
```

Comment:
```suggestion
import GETAllTrackingPreferences from './SettingsService.js'; 

```
should be in line with @annie951977's code. Just add that and I think its ok if u merge before her